README.md

```diff
@@ -24,18 +24,19 @@ use the `any http method` [suspend](https://kotlinlang.org/docs/reference/corout
 
 ```kotlin
 runBlocking {
-    val string = Fuel.get("https://publicobject.com/helloworld.txt").body.string()
+    val string: String = Fuel.get("https://publicobject.com/helloworld.txt").body.string()
```

Comment:
[q] Do we need to specify type string here?

Reply:
Something about IntelliJ idea was having trouble to understand the types of it.

Reply:
Small concern but it should be okay^^